bootstrap/terraform/aws-bootstrap/main.tf

```diff
@@ -78,17 +65,21 @@ module "single_az_node_groups" {
   ]
 }
 
-# module "multi_az_node_groups" {
-#   source                 = "github.com/pluralsh/module-library//terraform/eks-node-groups/multi-az-node-groups?ref=aws-multi-az"
-#   cluster_name           = var.cluster_name
-#   default_iam_role_arn   = module.cluster.worker_iam_role_arn
-#   tags                   = {}
-#   node_groups_defaults   = var.node_groups_defaults
-
-#   node_groups            = var.multi_az_node_groups
-#   set_desired_size       = false
-#   private_subnet_ids        = module.vpc.worker_private_subnets
-# }
+module "multi_az_node_groups" {
+  source                 = "github.com/pluralsh/module-library//terraform/eks-node-groups/multi-az-node-groups?ref=aws-multi-az"
+  cluster_name           = var.cluster_name
+  default_iam_role_arn   = module.cluster.worker_iam_role_arn
+  tags                   = {}
+  node_groups_defaults   = var.node_groups_defaults
+
+  node_groups            = var.multi_az_node_groups
+  set_desired_size       = false
+  private_subnet_ids     = module.vpc.worker_private_subnets_ids
+
+  ng_depends_on = [
+    module.cluster.config_map_aws_auth
+  ]
+}
 
 resource "aws_eks_addon" "vpc_cni" {
```

Comment:
we probably need to rewire the depends_on fields for the aws_eks_addons here to point to the new node group modules.